src/main/java/com/manardenza/entity/Hotel.java

```diff
@@ -0,0 +1,20 @@
+package com.manardenza.entity;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class Hotel extends AbstractObject {
+
+    private String name;
+    private String city;
+    private List<Room> rooms;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "rooms" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)